docs/guides/slp-software-setup.md

```diff
@@ -0,0 +1,367 @@
+const BITBOX = new BITBOXSDK.BITBOX({ restURL: 'https://rest.bitcoin.com/v2/' });
+const fundingAddress           = "simpleledger:qzvjfs5tuw56est4mwtzw3fegucgdwr00v0nkx3qwh";  // <-- must be simpleledger format
+const fundingWif               = "L1aeBNDeQ43PtFN2Jc9tbeqhNiqBgDHbxMctefHftHw4WtSo621e";     // <-- compressed WIF format
+const tokenReceiverAddress     = "simpleledger:qzvjfs5tuw56est4mwtzw3fegucgdwr00v0nkx3qwh";  // <-- must be simpleledger format
+const bchChangeReceiverAddress = "simpleledger:qzvjfs5tuw56est4mwtzw3fegucgdwr00v0nkx3qwh";  // <-- cashAddr or slpAddr format
+// For unlimited issuance provide a "batonReceiverAddress"
+const batonReceiverAddress     = "simpleledger:qzvjfs5tuw56est4mwtzw3fegucgdwr00v0nkx3qwh";
+
+// FOR TESTNET UNCOMMENT
+// const BITBOX = new BITBOXSDK.BITBOX({ restURL: 'https://trest.bitcoin.com/v2/' });
+// const fundingAddress           = "slptest:qpwyc9jnwckntlpuslg7ncmhe2n423304ueqcyw80l";
+// const fundingWif               = "cVjzvdHGfQDtBEq7oddDRcpzpYuvNtPbWdi8tKQLcZae65G4zGgy";
+// const tokenReceiverAddress     = "slptest:qpwyc9jnwckntlpuslg7ncmhe2n423304ueqcyw80l";
+// const bchChangeReceiverAddress = "slptest:qpwyc9jnwckntlpuslg7ncmhe2n423304ueqcyw80l";
+// // For unlimited issuance provide a "batonReceiverAddress"
+// const batonReceiverAddress     = "slptest:qpwyc9jnwckntlpuslg7ncmhe2n423304ueqcyw80l";
+
+const bitboxNetwork = new slpjs.BitboxNetwork(BITBOX);
+
+
+// 1) Select decimal precision for this new token
+let decimals = 2;
+let name = "My first awesome SLPJS Token";
+let ticker = "SLPJS";
+let documentUri = "info@simpleledger.io";
+let documentHash = null
+let initialTokenQty = 1000000
+
+let genesisTxid;
+
+// 2) Calculate the token quantity with decimal precision included
+initialTokenQty = (new BigNumber(initialTokenQty)).times(10**decimals);
+
+
+// 3) Get all balances at the funding address.
+let balances; 
+(async function() {
+  balances = await bitboxNetwork.getAllSlpBalancesAndUtxos(fundingAddress);
+  console.log("'balances' variable is set.");
+  console.log('BCH balance:', balances.satoshis_available_bch);
+
+// 4) Set private keys
+  balances.nonSlpUtxos.forEach(txo => txo.wif = fundingWif)
+  
+// 5) Use "simpleTokenGenesis()" helper method
+  genesisTxid = await bitboxNetwork.simpleTokenGenesis(
+        name, 
+        ticker, 
+        initialTokenQty,
+        documentUri,
+        documentHash,
+        decimals,
+        tokenReceiverAddress,
+        batonReceiverAddress,
+        bchChangeReceiverAddress,
+        balances.nonSlpUtxos
+        )
+    console.log("GENESIS txn complete:",genesisTxid)
+})();
+```
+
+Here you’ll want to make sure to change the `fundingAddress`, `fundingWif`, `tokenReceiverAddress`, `bchChangeReceiverAddress` and `batonReceiverAddress` in the script to the ones you obtained in the previous steps. Also, remember to send a small amount of BCH to the funding address using the cashAddr format. Some cents will be enough.
```

Comment:
doesn't matter if its sent to cashAddr or simpleledger format for the address, they represent the same scripthash

Reply:
Yeah, I was going for the case where they try to fund using a non-slp wallet which doesn't accept simpleledger format to send the funds. I'll clarify.
Fixed.